app/utils/cma-hearing-details.ts

```diff
@@ -0,0 +1,36 @@
+import { Request } from 'express';
+import moment from 'moment';
+import { dayMonthYearFormat, timeFormat } from './date-utils';
+
+function getHearingDate(req: Request) {
+  let formattedDeadline;
+
+  const dueDate = req.session.appeal.listCmaHearingDate;
+  formattedDeadline = moment(dueDate).format(dayMonthYearFormat);
+  return formattedDeadline;
+}
+
+function getHearingTime(req: Request) {
+  let hearingTime;
+
+  const dueDate = req.session.appeal.listCmaHearingDate;
+  hearingTime = moment(dueDate).format(timeFormat);
+  return hearingTime;
+}
+
+function getHearingCentre(req: Request) {
+  let hearingCentre;
+
+  const name = req.session.appeal.listCmaHearingCentre;
+  hearingCentre = name.replace(/([a-z0-9])([A-Z])/g, '$1 $2')
```

Comment:
Instead of replace should we map the listCma into an enum type that way you can pull the Hearing centre easily